swagger.yaml

```diff
@@ -2182,12 +2313,62 @@ components:
           type: string
           format: date-time
           readOnly: true
-    
+
     DraftData:
       type: object
       additionalProperties:
         type: object
 
+    AuditLogEntry:
+      type: object
+      properties:
+        id:
+          type: string
+          format: uuid
+          readOnly: true
+          description: Entity identifier
+        userId:
+          type: string
+          format: uuid
+          readOnly: true
+          description: Id of creator user
+        replyId:
+          type: string
+          format: uuid
+          readOnly: true
+          description: Id of the reply
+        attachmentId:
+          type: string
+          format: uuid
+          readOnly: true
+          description: Id of the attachment
+        time:
```

Comment:
Should be renamed to createdAt for keeping the resource consistent with other resources